_posts/20-01-22-Citrix-en-wildcards.md

```diff
@@ -16,7 +16,7 @@ Meer informatie en de laatste stand van zaken met betrekking tot deze case is te
 
 ### <a name="english"></a>English
 
-Of the 700 hunderd systems we identified as being vulnerable on the night from 9th to 10th of January, over 450 systems were secured using wildcard certificates. Theft of these certificates and their keys is not just a risk for the Citrix environment, but it allows an attacker to intercept TLS secured connections to any subdomain of the organisation without the user being notified. 
+Of the 700 hunderd systems we identified as being vulnerable, over 450 systems were secured using wildcard certificates. Theft of these certificates and their keys is not just a risk for the Citrix environment, but it allows an attacker to intercept TLS secured connections to any subdomain of the organisation without the user being notified. 
```

Comment:
Dit ging echt specifiek over systemen die in de in de nacht van 9 op 10 januari kwetsbaar waren.